### ISSUE TITLE
Run Typhoon from Matlab command window

### ISSUE THREAD

**Issue by RadivT** [MaritiemUGent/Typhoon#4]
Hi, Typhoon seems to be a very nice software! I just came over it. I see there is an executable, but is it possible to run Typhoon from Matlab command window without the executable? Are all the necessary .m files present in the Github download?

Thanks!

**Comment by MTFleet** [comment]
yes, you can run it in Matlab directly, all files are available here

**Comment by gshamov** [comment]
Is there any way to create a Linux executable if one has Linux Matlab? Thanks!

**Comment by MTFleet** [comment]
No but you can run it in Matlab directly without the executable
